src/Mvc/src/Microsoft.AspNetCore.Mvc/Microsoft.AspNetCore.Mvc.csproj

```diff
@@ -9,7 +9,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.AspNetCore.Mvc.Analyzers\Microsoft.AspNetCore.Mvc.Analyzers.csproj" PrivateAssets="None" />
```

Comment:
Analyzers don't make their way into the shared framework anymore anyhow. Having this caused assembly load issues because the functional tests would attempt to load the analyzers project which in turn explode at runtime because there's no Roslyn. This is a more significant issue that we may want to document. Up for suggestions here.